### ISSUE TITLE
Astra DB - Connectivity warning message keeps printing

### ISSUE THREAD

**Issue by kathirsvn** [JanusGraph/janusgraph#2945]
- Version: 0.6.0
- Storage Backend: cql
- Mixed Index Backend: none
- Link to discussed bug: none
- Expected Behavior: No error/warning message should be printed 
- Current Behavior: Getting 'Error while opening new channel' warning printed every few seconds while connecting to the Astra DB as per the documentation
- Steps to Reproduce: Create an Astra DB and connect to it from JanusGraph as per the documentation

### Log that repeats every few seconds

```
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-6] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.216.4:29042]  Error while opening new channel (ConnectTimeoutException: connection timed out: 172.25.216.4/172.25.216.4:29042)
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-6] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.216.4:29042]  Error while opening new channel (ConnectTimeoutException: connection timed out: 172.25.216.4/172.25.216.4:29042)
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-6] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.216.4:29042]  Error while opening new channel (ConnectTimeoutException: connection timed out: 172.25.216.4/172.25.216.4:29042)
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-6] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.216.4:29042]  Error while opening new channel (AnnotatedConnectException: Connection refused: 172.25.216.4/172.25.216.4:29042)
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-8] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.142.4:29042]  Error while opening new channel (ConnectTimeoutException: connection timed out: 172.25.142.4/172.25.142.4:29042)
2022-01-16 04:45:48.995  WARN 1380 --- [Session-admin-6] c.d.o.d.internal.core.pool.ChannelPool   : [JanusGraph Session|/172.25.216.4:29042]  Error while opening new channel (ConnectTimeoutException: connection timed out: 172.25.216.4/172.25.216.4:29042)

```

**Comment by kathirsvn** [comment]
Created a [PR](https://github.com/JanusGraph/janusgraph/pull/2946) fixing the documentation for this, waiting for my CLA approval

**Comment by FlorianHockmann** [comment]
Fixed in #2953.
